docs/developers/define-secrets.rst

```diff
@@ -261,6 +261,60 @@ To sync multiple environments at once:
 
    ./update_all_secrets.sh
 
+Create static secrets in 1Password (new)
+========================================
+
+.. note::
+
+   This section only applies to Phalanx environments run by SQuaRE that have been converted to the new 1Password setup that is currently under development.
+
+For SQuaRE-run Phalanx environments, static secrets for applications are stored in a 1Password vault before being automatically synced to the Vault service.
+Such secrets are things for external cloud services where we don't automatically provision accounts and password.
+When we manually create such a secret, we store it in 1Password.
+
+This step may have to be done for you by a Phalanx environment administrator depending on how permissions in Vault and any underlying secrets store are handled for your environment.
+
+.. note::
+
+   This document only covers creating a 1Password-backed secret for the first time for an application.
+   If you want to update a secret, either by adding new 1Password secrets or by changing their secret values, you should follow the instructions in :doc:`/developers/update-a-onepassword-secret`.
+
+.. warning::
+
+   This is the new process for storing secrets in 1Password that is currently under development.
+   Most environments should continue to use the instrutions in :ref:`dev-add-onepassword`.
```

Comment:
```suggestion
   Most environments should continue to use the instructions in :ref:`dev-add-onepassword`.
```

Reply:
More generally, I think this `.. warning::` should probably be rolled up with the `.. note::` at the beginning of the section. We don't want to overwhelm with extra admonitions.